lib/commands/xctest.js

```diff
@@ -0,0 +1,87 @@
+import B from 'bluebird';
+import { logger } from 'appium-support';
+import log from '../logger';
+import appManagement from './app-management';
+
+let helpers = {}, commands = {}, extensions = {};
+
+const xctestLog = logger.getLogger('XCTest');
+
+function assertIDB (context) {
+  if (!context.opts?.device?.idb) {
+    log.errorAndThrow(`To use XCTest runner, IDB (https://github.com/facebook/idb) must be installed ` +
+      `and sessions must be run with the "launchWithIDB" capability`);
+  }
```

Comment:
this method could return `opts` after validation, so it could be chained

Reply:
Better yet I'll make it return IDB.